tests/transaction_tests/main.rs

```diff
@@ -0,0 +1,349 @@
+/// Geth accepts this tx as valid, I can't determine what critera by which 137 is an invalid chain id
+///
+/// RLPIncorrectByteEncoding127
+/// RLPIncorrectByteEncoding00
+/// RLPIncorrectByteEncoding01
+/// These test cover byte encoding which follows the general spirit of the RLP standard but are wonky in some way, normally
+/// this means specifying a single byte int as a multi byte value and padding with zeros or some other encoding variant that
+/// is arguably correct but should be forbidden to ensure things are always represented cannonically. In our case these are
+/// quite hard to detect due to the use of serde in our deserialization library, the abstraction between decoding the length
+/// and the actual data means we never have the right data in the same context to detect these exceptions. Fixing this would
+/// require a major refactor of the deserialization engine to avoid the use of serde, which would probably be a good idea.
+const BLACKLISTED_TESTS: [&'static str; 4] = [
+    "tr201506052141PYTHON",
+    "RLPIncorrectByteEncoding127",
+    "RLPIncorrectByteEncoding01",
+    "RLPIncorrectByteEncoding00",
+];
+
+fn test_on_blacklist(test_name: &str) -> bool {
+    for t in BLACKLISTED_TESTS {
+        if t == test_name {
+            return true;
+        }
+    }
+    false
+}
+
+fn visit_dirs(dir: &Path, cb: &mut dyn FnMut(&DirEntry)) -> io::Result<()> {
+    if dir.is_dir() {
+        for entry in fs::read_dir(dir)? {
+            let entry = entry?;
+            let path = entry.path();
+            if path.is_dir() {
+                visit_dirs(&path, cb)?;
+            } else {
+                cb(&entry);
+            }
+        }
+    }
+    Ok(())
+}
+
+fn get_repo_path() -> PathBuf {
+    Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf()
+}
+
+fn get_fixtures_path() -> PathBuf {
+    let mut path = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf();
+    path.push("tests");
+    path.push("fixtures");
+    path
+}
+
+/// Loads a single test fixture along with it's name
+fn load_fixture(path: &Path) -> HashMap<String, TestFixture> {
+    // Read JSON in advance before running this particular test.
+    // This way we can construct human readable test name based on the JSON contents, and
+    let file = File::open(path).unwrap_or_else(|_| panic!("Could not open file {path:?}"));
+    let buffered_reader = BufReader::new(file);
+    let json_data: Value =
+        serde_json::from_reader(buffered_reader).expect("Unable to read JSON file");
+    // Deserialize fixture object
+    serde_json::from_value(json_data).unwrap()
+}
+
+/// Loads the transaction and test data for a single fixture along with it's name
+fn load_filler(fixture: &TestFixture) -> HashMap<String, TestFiller> {
+    // Load filler
+    let mut filler_path = get_fixtures_path();
+    filler_path.push(&fixture.info.source);
+    // due to an error in the test files eip1559 fillers all have an incorrect path of ttValue, this hotpatches that issue
```

Comment:
This can be removed now, right? I saw your PR was merged.

Reply:
good call will try removing it